### ISSUE TITLE
Pointing my own domain says "No such app"

### ISSUE THREAD

**Issue by sakydev** [HermanMartinus/bearblog#76]
I pointed a TLD on my bear blog and it keeps saying "No such app" on a default Heroku page.

You can see it here http://www.sakytalks.com/
My blog URL is https://sakytalks.bearblog.dev/

Please let me know if more information is needed on my end

**Comment by HermanMartinus** [comment]
Hi there, it looks like you hadn’t added the domain on the dashboard. I’ve fixed it and your site is now available at www.sakytalks.com <http://www.sakytalks.com/>

All the best, 
Herman

**Comment by sakydev** [comment]
Hi @HermanMartinus ,

Thank you so much. It works now. I tried it before and the domain was added successfully but it kept giving this error. Then I came across this 

https://stackoverflow.com/questions/53026340/herokudns-with-custom-domain-no-such-app

The problem here was that I added the domain as it says in "eg: 'example.com'" in Bearblog. I think it works only when you add it with "www.example.com" as you've done now.
